raccoon_src/utils/request_handler.py

```diff
@@ -91,12 +90,9 @@ def send(self, method="GET", *args, **kwargs):
         proxies = self._get_request_proxies()
 
         try:
-            if method.lower() == "get":
-                return requests.get(proxies=proxies, headers=self.headers, cookies=self.cookies, *args, **kwargs)
-            elif method.lower() == "post":
-                return requests.post(proxies=proxies, headers=self.headers, cookies=self.cookies, *args, **kwargs)
-            elif method.lower() == "head":
-                return requests.head(proxies=proxies, headers=self.headers, cookies=self.cookies, *args, **kwargs)
+            if method.upper() in ['GET', 'POST', 'HEAD']:
+                kwargs['timeout'] = kwargs['timeout'] if 'timeout' in kwargs else 5
```

Comment:
you have couple of usages that set timeout, to support them and avoid `TypeError: got multiple values for keyword argument 'timeout'` 
BTW, TypeError should renamed  to MultipleArgumentValuesError 
I check if timeout is set, if not, default 5, read the review before to understand why it's a constant